storm-core/src/storm.thrift

```diff
@@ -193,6 +193,7 @@ struct ExecutorStats {
   1: required map<string, map<string, i64>> emitted;
   2: required map<string, map<string, i64>> transferred;
   3: required ExecutorSpecificStats specific;
+  4: required double rate;
```

Comment:
Having this be required technically breaks compatibility.  I'm not sure if this is a problem or not.  ExecutorStats is exposed to the end user, but really only as a consumer. So it makes it impossible for a newer client to interact with an older nimbus instance.  I think this is OK, but we should be very careful about these kinds of things going forward. 

Have you tested using older clients with a newer nimbus instance?